CHANGELOG.md

```diff
@@ -65,7 +71,7 @@ TL;DR: Store LRU cache is no longer leaking, Upgraded Thanos UI to Prometheus 2.
 
 ## Deprecated 
 
-- [#1008](https://github.com/improbable-eng/thanos/pull/1008) Removed Gossip implementation.
+- [#1008](https://github.com/improbable-eng/thanos/pull/1008) *breaking* Removed Gossip implementation.
```

Comment:
Maybe add that we removed that flag too?

Reply:
done